Services/FileService/Startup.cs

```diff
@@ -38,6 +43,35 @@ public void ConfigureServices(IServiceCollection services)
             ConfigureMappers(services);
             ConfigureRepositories(services);
             ConfigureValidators(services);
+            ConfigureRabbitMq(services);
+
+            services.AddMassTransitHostedService();
+        }
+
+        private void ConfigureRabbitMq(IServiceCollection services)
+        {
```

Comment:
I think this is wrong naming. I use "ConfigureMassTransit" and this should be the correct name since we really do configure MassTransit, don't we?

Reply:
fixed